tools/hooks/install.bat

```diff
@@ -1,2 +0,0 @@
-@call "%~dp0\..\bootstrap\python" -m hooks.install %*
```

Comment:
why removing this?

Reply:
https://github.com/discordia-space/CEV-Eris/pull/6910

Reply:
@LetterN Do you think removing this file could break things? It seems to be a duplicate of Install.bat and to not be called in the code but maybe we're missing something, since you made the PR that set that up.

Reply:
Removed via another PR.